### ISSUE TITLE
Corrupt files with 0 size in the dataset

### ISSUE THREAD

**Issue by IgorSusmelj** [alexeygrigorev/clothing-dataset#1]
Hey @alexeygrigorev, there are a few images in the dataset which have 0 size and won't load properly:

- 040d73b7-21b5-4cf2-84fc-e1a80231b202.jpg
- 1d0129a1-f29a-4a3f-b103-f651176183eb.jpg
- 784d67d4-b95e-4abb-baf7-8024f18dc3c8.jpg
- b72ed5cd-9f5f-49a7-b12e-63a078212a17.jpg
- c60e486d-10ed-4f64-abab-5bb698c736dd.jpg
- d028580f-9a98-4fb5-a6c9-5dc362ad3f09.jpg

**Comment by shruti-shyam** [comment]
Quick fix:
```
from os import listdir
from PIL import Image

image_folder = 'path/to/clothing-dataset/images/'

for image_name in listdir(image_folder):
    image_path = image_folder + image_name
    try:
        img = Image.open(image_path)
        img.verify()     # to verify if its an img
        img.close()      #to close img and free memory space
    except (IOError, SyntaxError) as e:
        print('Bad file:', image_path, e)
        os.remove(image_path)
```

**Comment by alexeygrigorev** [comment]
Oh sorry for the silence from my side. Thanks @shruti-shyam for the snippet.
I think it's actually fine just to delete them. I will probably do that at some point, but if you can create a PR with that, it will be more than welcome

**Comment by alexeygrigorev** [comment]
Resolved in #3
